osu.Game.Resources/Shaders/sh_ArgonBarPathBackground.fs

```diff
@@ -0,0 +1,50 @@
+#ifndef ARGON_BAR_PATH_BACKGROUND_FS
+#define ARGON_BAR_PATH_BACKGROUND_FS
```

Comment:
I don't believe these definitions are necessary for concrete shaders. They are defined for all header files to avoid a shader file indirectly including a header file more than once.

Reply:
These were added to each shader (https://github.com/ppy/osu-framework/pull/5832) so I'm following along

Reply:
Right, my bad on this one. I was checked out on an ancient version of master and couldn't find these definitions framework-side.